### PR TITLE
tests/thread_basic: rename python testrunner script

### PR DESCRIPTION
**Pull request by aabadie**
Use consistent name for thread_basic python test script.

### CODE REVIEW

File: tests/thread_basic/tests/01-run.py [hunk 1]
```diff
@@ -12,6 +12,7 @@
 sys.path.append(os.path.join(os.environ['RIOTBASE'], 'dist/tools/testrunner'))
 import testrunner
 
+
```
Comment (smlng) [inline tests/thread_basic/tests/01-run.py]: unrelated and not needed?

Comment (aabadie) [reply tests/thread_basic/tests/01-run.py]: Well, that just a fix for a small pep8

**Comment by aabadie** [comment]
Another "easy" one with the testrunner

File: tests/thread_basic/tests/01-run.py (hunk 1 above)
Comment (cladmi) [reply tests/thread_basic/tests/01-run.py]: I think that fixing pep8 could be done in a different commit.

Comment (aabadie) [reply tests/thread_basic/tests/01-run.py]: Ok ;)

Comment (aabadie) [reply tests/thread_basic/tests/01-run.py]: Done

**Comment by aabadie** [comment]
Rebased to latest master to take into account #7953 and fix CI

**Pull request merged**